aws-services/uninstall-aws-services.hbs.md

```diff
@@ -0,0 +1,104 @@
+# Uninstall AWS Services
+
+This topic tells you how to uninstall the AWS Services package from the Tanzu Application Platform (commonly
+known as TAP) package repository.
+
+## <a id="prepare-infra"></a> Step 1: Prepare your infrastructure
+
+Before uninstalling the package, you will have to decide what to do with the existing resources you have
+claimed.
+
+* In case you want to keep a resource:
+
+  1. Identify the corresponding Crossplane managed resource:
+
+      ```console
+      kubectl get instances.rds.aws.upbound.io
+      ```
+
+  1. Edit the resource:
+
+      ```console
+      kubectl edit instances.rds.aws.upbound.io <RESOURCE-NAME>
+      ```
+
+  1. Change the resource `deletionPolicy` to `Orphan`:
+
+      ```yaml
+      apiVersion: rds.aws.upbound.io/v1beta1
+      kind: Instance
+      metadata:
+        # ...
+      spec:
+        deletionPolicy: Orphan
+        # ...
+      ```
+
+* In case you want to get rid of a resource:
+
+  1. Delete the corresponding claim:
+
+      ```console
+      tanzu service class-claim delete <CLAIM-NAME>
+      ```
+
+  1. Wait for the resource to disappear in the AWS console.
```

Comment:
If users don't delete the class claim and proceed to step 2. What will happen? Just wondering if we should add a sentence to describe that scenario or not.

Reply:
If you neither delete the claim nor set the resource to `Orphan`, resources will still survive in AWS but more stuff will be left behind in the cluster (what I saw was the `Instance` + its corresponding CRD).

The reason I don't want to recommend this is that this seems like a lucky accident: the controller gets uninstalled _before_ the `Instance` can be finalised, but I'm not sure how much of a coincidence this is. In the unfortunate event of the `Instance` being deleted before the controller, there's a chance that resources on AWS would be deleted (because of `deletionPolicy: Delete`). I've never seen this in my tests, but I can't explain why.

Reply:
If you haven't already, perhaps it's worth mentioning something along the line of "if you do not delete class claims before uninstalling, it could leave your TAP and AWS in a dirty state that prevents reinstalling the package..." What do you think?

Reply:
Yeah makes sense. I have something similar for the `ProviderConfig` already.